lms/envs/common.py

```diff
@@ -1917,6 +1922,15 @@
 CERT_NAME_SHORT = "Certificate"
 CERT_NAME_LONG = "Certificate of Achievement"
 
+#################### Badgr OpenBadges generation #######################
+BADGR_API_TOKEN = None
+BADGR_BASE_URL = "http://localhost:8005"
+BADGR_ISSUER_SLUG = "example-issuer"
+# A dictionary of course modes to file paths. Example:
+# {'honor': '/path/to/badge_image.png'}
+# Badge images must be square and in PNG or SVG format.
+BADGR_IMAGE_SOURCES = {}
```

Comment:
@antoviaque What if there were default images that wouldn't need to be specially uploaded? Or if we used URLs instead, and the image would have to be downloaded? Since the download action would only have to be done upon the first badge generation (barring some slight race condition weirdness that shouldn't cause too much of a problem in practice), this might be an easier method.